AppcuesCapacitor.podspec

```diff
@@ -13,6 +13,6 @@ Pod::Spec.new do |s|
   s.source_files = 'ios/Plugin/**/*.{swift,h,m,c,cc,mm,cpp}'
   s.ios.deployment_target  = '13.0'
   s.dependency 'Capacitor'
-  s.dependency 'Appcues', '~> 1.1.1'
+  s.dependency 'Appcues', '~> 1.2'
```

Comment:
note: in the example app in this repo, the podfile.lock is excluded in .gitignore, thus no change visible here.